Doesn't run with JDK13

When trying to run the `pingCmd` it fails with the exception:
```
Exception in thread "main" javax.ws.rs.ProcessingException: java.net.SocketException: Cannot find method "setCreated" in java.net.Socket. Unsupported JVM?
        at org.glassfish.jersey.apache.connector.ApacheConnector.apply(ApacheConnector.java:496)
        at org.glassfish.jersey.client.ClientRuntime.invoke(ClientRuntime.java:278)
        at org.glassfish.jersey.client.JerseyInvocation.lambda$invoke$0(JerseyInvocation.java:753)
        at org.glassfish.jersey.internal.Errors.process(Errors.java:316)
        at org.glassfish.jersey.internal.Errors.process(Errors.java:298)
        at org.glassfish.jersey.internal.Errors.process(Errors.java:229)
        at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:414)
        at org.glassfish.jersey.client.JerseyInvocation.invoke(JerseyInvocation.java:752)
        at org.glassfish.jersey.client.JerseyInvocation$Builder.method(JerseyInvocation.java:419)
        at org.glassfish.jersey.client.JerseyInvocation$Builder.get(JerseyInvocation.java:319)
        at com.github.dockerjava.jaxrs.PingCmdExec.execute(PingCmdExec.java:24)
        at com.github.dockerjava.jaxrs.PingCmdExec.execute(PingCmdExec.java:11)
        at com.github.dockerjava.jaxrs.AbstrSyncDockerCmdExec.exec(AbstrSyncDockerCmdExec.java:23)
        at com.github.dockerjava.core.command.AbstrDockerCmd.exec(AbstrDockerCmd.java:35)
        at Main.dockerJava(Main.java:62)
        at Main.main(Main.java:116)
Caused by: java.net.SocketException: Cannot find method "setCreated" in java.net.Socket. Unsupported JVM?
        at org.newsclub.net.unix.NativeUnixSocket.setCreated(Native Method)
        at org.newsclub.net.unix.AFUNIXSocket.setIsCreated(AFUNIXSocket.java:54)
        at org.newsclub.net.unix.AFUNIXSocket.<init>(AFUNIXSocket.java:48)
        at org.newsclub.net.unix.AFUNIXSocket.newInstance(AFUNIXSocket.java:77)
        at org.newsclub.net.unix.AFUNIXSocket.newInstance(AFUNIXSocket.java:72)
        at com.github.dockerjava.jaxrs.ApacheUnixSocket.<init>(ApacheUnixSocket.java:51)
        at com.github.dockerjava.jaxrs.UnixConnectionSocketFactory.createSocket(UnixConnectionSocketFactory.java:66)
        at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:119)
        at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:373)
        at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:394)
        at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
        at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
        at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
        at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
        at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
        at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:72)
        at org.glassfish.jersey.apache.connector.ApacheConnector.apply(ApacheConnector.java:450)
        ... 15 more
```

--- a/pom.xml
+++ b/pom.xml
@@ -68,7 +68,7 @@
 		<slf4j-api.version>1.7.25</slf4j-api.version>
 
 		<bouncycastle.version>1.60</bouncycastle.version>
-		<junixsocket.version>2.2.0</junixsocket.version>
+		<junixsocket.version>2.2.1</junixsocket.version>
 		<guava.version>19.0</guava.version>
 
 		<!-- test dependencies -->

